content/en/service_management/case_management/troubleshooting.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Troubleshooting
+kind: Documentation
+---
+
+## Overview
+
+This guide is intended to help you resolve issues with third-party integrations in Case Management. If you continue to have trouble, reach out to [Datadog support][1] for further assistance.
+
+## Jira
+
+Jira issue types with custom fields, private Jira projects, and on-premises Jira instances are not supported. If you are having trouble with automatic Jira ticket creation with syncing, see the following sections:
+
+### Configuration
+
+1. If Jira projects are not populating the dropdown on the Jira integration configuration screen, check that you have the `manage_integrations` permission. 
+
+1. Ensure that you have configured a webhook to receive events from Jira.
```

Comment:
Can we link out to the webhooks section of this PR? https://github.com/DataDog/dogweb/pull/110043/files The section is called "Configure a Jira webhook"

Both PRs should be merged at the same time